packages/web-components/src/components/password-input/password-input.stories.ts

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import View16 from '@carbon/icons/lib/view/16.js';
+import FolderOpen16 from '@carbon/icons/lib/folder--open/16.js';
+import Folders16 from '@carbon/icons/lib/folders/16.js';
+import './index';
+import '../form/form-item';
+import '../icon-button';
+import {
+  INPUT_SIZE,
+  INPUT_TOOLTIP_ALIGNMENT,
+  INPUT_TOOLTIP_DIRECTION,
+} from './password-input';
+
+const actions = html`
+  <cds-icon-button kind="ghost" slot="actions" size="lg">
+    ${View16({ slot: 'icon' })}
+    <span slot="tooltip-content"> View </span>
+  </cds-icon-button>
+  <cds-icon-button kind="ghost" slot="actions" size="lg">
+    ${FolderOpen16({ slot: 'icon' })}
+    <span slot="tooltip-content"> Open folder</span>
+  </cds-icon-button>
+  <cds-icon-button kind="ghost" slot="actions" size="lg">
+    ${Folders16({ slot: 'icon' })}
+    <span slot="tooltip-content"> Folders </span>
+  </cds-icon-button>
+`;
+
+const sizes = {
+  [`Small size (${INPUT_SIZE.SMALL})`]: INPUT_SIZE.SMALL,
+  [`Medium size (${INPUT_SIZE.MEDIUM})`]: INPUT_SIZE.MEDIUM,
+  [`Large size (${INPUT_SIZE.LARGE})`]: INPUT_SIZE.LARGE,
+};
+
+const args = {
+  defaultWidth: 300,
+  disabled: false,
+  helperText: 'Helper text',
+  hideLabel: false,
+  hidePasswordLabel: 'Hide Password',
+  inline: false,
+  invalid: false,
+  invalidText: 'Error message goes here',
+  labelText: 'Label text',
+  placeholder: 'Placeholder text',
+  readonly: false,
+  showPasswordLabel: 'Show Password',
+  size: INPUT_SIZE.MEDIUM,
+  tooltipAlignment: INPUT_TOOLTIP_ALIGNMENT.CENTER,
```

Comment:
React default it to "end"
<img width="334" alt="image" src="https://github.com/user-attachments/assets/099f3202-6504-4b61-9d60-27799f007f70" />

```suggestion
  tooltipAlignment: INPUT_TOOLTIP_ALIGNMENT.END,
```